frontend/src/layout/navigation-3000/sidepanel/panels/activation/activationLogic.ts

```diff
@@ -65,8 +65,6 @@ export const activationLogic = kea<activationLogicType>([
             ['loadPluginsSuccess', 'loadPluginsFailure'],
             sidePanelStateLogic,
             ['openSidePanel'],
-            eventUsageLogic,
-            ['reportActivationSideBarShown'],
```

Comment:
This was removed in a previous PR and isn't call anymore.

Reply:
should we call this to know how often it's shown?